### DISCUSSION TITLE
Sending funds from Trinity wallet (logged in after long absence)

### DISCUSSION THREAD

**Discussion by Voytoos** [iotaledger/firefly#8553] [Help & Support]
I tried to log into my Trinity wallet to check my IOTA funds. I found out that Trinity is no longer working. I've been trying to send funds to another wallet but there is this notification still popping up "The node returned an error while fetching your account information. Try changing node." I tried to change the node manually, but whichever I choose from the list, the notification keeps coming. 
How do I get my funds transferred from Trinity now? Thanks in advance for your help.

**Comment by no8ody** [comment]
You need to migrate your tokens, please read the following blog post: https://blog.iota.org/iota-legacy-migration-tool/
